### PR TITLE
refactor: small edits

### PR DESCRIPTION
**Pull request by witgaw**

### CODE REVIEW

File: docs/concepts/trading-on-vega/positions-margin.md [hunk 1]
```diff
@@ -3,47 +3,42 @@ sidebar_position: 2
 title: Positions and margin
 hide_title: false
 ---
-Trading margined derivatives (such as futures) allows you to create leveraged positions, meaning you only need some of the notional value of a trade to place orders on a market on Vega. 
+Trading margined derivatives (such as futures) allows you to create leveraged positions, meaning you only need some of the notional value of an instrument to place orders on a market on Vega. 
 
-Leverage is, in effect, the inverse of the initial margin that the protocol takes from your collateral when you place a trade.
```
Comment (candida-d) [inline docs/concepts/trading-on-vega/positions-margin.md:8]: I don't mind if you reject this sentence specifically, but I did want to include at least a nod to the question of "how does leverage work on Vega?" so I'd be very keen for an alternative :)

File: docs/concepts/trading-on-vega/positions-margin.md [hunk 2]
```diff
@@ -3,47 +3,42 @@ sidebar_position: 2
...
 
 When a party on Vega opens a position, the minimum amount of assets required to open that position is put into a margin account for that party in that market. 
 
-Over the course of the position's lifetime, the margin requirements will likely change - the margin account may be topped up, and/or some margin is released back to collateral. If that party is trading on more than one market that uses the same asset, the collective positions on those markets will inform how much is set aside for margin. Margin balances are also affected by unrealised profit and loss. 
+Over the course of the position's lifetime, the margin requirements will likely change - the margin account may be topped up, and/or some margin is released back to collateral. If that party is trading on more than one market that uses the same asset, the collective positions on those markets will inform how much is set aside for margin. Margin balances are also affected by the price movements in the market where the position is being held. These hypothetical changes in position's value are called  unrealised profit and loss.
```
Comment (candida-d) [inline docs/concepts/trading-on-vega/positions-margin.md:17]:
```suggestion
Over the course of the position's lifetime, the margin requirements will likely change - the margin account may be topped up, and/or some margin is released back to collateral. If that party is trading on more than one market that uses the same asset, the collective positions on those markets will inform how much is set aside for margin. Margin balances are also affected by the price movements in the market where the position is being held. These hypothetical changes in a position's value are called unrealised profit and loss. 
```

File: docs/concepts/trading-on-vega/positions-margin.md [hunk 3]
```diff
@@ -3,47 +3,42 @@ sidebar_position: 2
...
-
-When marking to market, the protocol takes the current market price and recalculates traders' margin requirements based on how their position is affected by price moves.
+Marking to market refers to settling gains and losses due to changes in the market value. Marking to market aims to provide a realistic appraisal of a position based on the current market conditions.
 
-If the market price goes up, a trader that holds a long position receives money in their margin account – equal to the underlying's change in value – from a trader that holds a short position, and conversely if the value goes down, the holder of the short position receives money from the holder of the long position.
+If the market price goes up, traders that hold a long positions receive money into their margin account – equal to the change in the notional value of their positions – from traders that hold short positions, and conversely if the value goes down, the holders of the short positions receive money from the holders of long positions.
```
Comment (candida-d) [inline docs/concepts/trading-on-vega/positions-margin.md:31]:
```suggestion
If the market price goes up, traders that hold a long position receive money into their margin account – equal to the change in the notional value of their positions – from traders that hold short positions, and conversely if the value goes down, the holders of the short positions receive money from the holders of long positions.
```

**Review by candida-d — COMMENTED** [review]
Looks great overall! Thanks for your updates. Minor fixes plus one request

File: docs/concepts/trading-on-vega/positions-margin.md (hunk 1 above)
Comment (witgaw) [reply docs/concepts/trading-on-vega/positions-margin.md:8]: It's just that it's inaccurate, and I though "Leverage, meanwhile, describes how many times larger is the notional value of that position compared to the margin you have dedicated to it." below does the trick.

File: docs/concepts/trading-on-vega/positions-margin.md (hunk 3 above)
Comment (witgaw) [reply docs/concepts/trading-on-vega/positions-margin.md:31]: Modified now to "trader that hold long positions". I wanted to indicate that it's not a 1 to 1 flow, we just collect money from all those who made a loss and distribute to among those that made a profit, but it's not like people are paired in any specific way.

Comment (candida-d) [reply docs/concepts/trading-on-vega/positions-margin.md:31]: Sounds good - I should have commented on this one as what I want to fix is the plural/singular agreement, but I don't mind how it's done, as long as they agree.

File: docs/concepts/trading-on-vega/positions-margin.md (hunk 1 above)
Comment (candida-d) [reply docs/concepts/trading-on-vega/positions-margin.md:8]: Great, that works, thank you!

File: docs/concepts/trading-on-vega/positions-margin.md [hunk 4]
```diff
@@ -115,7 +112,7 @@ The release level is scaled from the *maintenance margin* amount. It's calculate
 `[release level] = [maintenance margin] x [collateral_release scaling factor]`
 
 ### Calculating the margin on open orders
-The network calculates the overall long / short position including the submitted order. Depending on which one is larger a long or short risk factor is used for margin calculation. The maintenance margin (for futures) is then a product of the largest position, the corresponding risk factor and the `mark price`. These capture the outcome of the probabilistic distribution of future market moves, and are market specific.
+The network calculates the overall long / short position including the submitted order. Depending on which one is larger a long or short risk factor is used for margin calculation. The maintenance margin (for futures) is then a product of the largest position, the corresponding risk factor and the `mark price`. Risk factirs capture the outcome of the probabilistic distribution of future market moves, and are market specific.
```
Comment (candida-d) [inline docs/concepts/trading-on-vega/positions-margin.md:115]:
```suggestion
The network calculates the overall long / short position including the submitted order. Depending on which one is larger a long or short risk factor is used for margin calculation. The maintenance margin (for futures) is then a product of the largest position, the corresponding risk factor and the `mark price`. Risk factors capture the outcome of the probabilistic distribution of future market moves, and are market specific.
```

**Review by candida-d — APPROVED** [review]

**Pull request merged**